.travis.yml

```diff
@@ -11,6 +11,7 @@ android:
     - extra-google-m2repository
     - extra-android-support
     - extra-google-google_play_services
-
+install :
+  - travis_wait 20
```

Comment:
The process to compile everything on the project take a lot of time and make travis fail. To resolve this problem i used this solution at the link : 
https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received